packages/sdk-v2/src/utils/swapHelper.ts

```diff
@@ -0,0 +1,9 @@
+export const swapHelper = async (account: string) => {
+  const tx = fetch(`https://good-server.herokuapp.com/verify/swaphelper`, {
+    method: "POST",
+    headers: { "Content-type": "application/json" },
+    body: JSON.stringify({ account })
+  });
+
+  return tx;
+};
```

Comment:
we could do that, but not right now. The BuyGd contract has not been released to the dev envs so all this flow would fail here:
https://github.com/GoodDollar/GoodServer/blob/af28e7f81a61e94e773b106b301cdd3592e0cb75/src/server/blockchain/Web3Wallet.js#L261

As the contract address based on this.network would fail

Reply:
the server link should still and not be hardcoded

Reply:
resolved in latest refactor by @sirpy

Reply:
no it wasn't resolved. you need to have the default according to the env